manifests/jobs-api.yaml

```diff
@@ -19,7 +19,7 @@ spec:
         - image: ghcr.io/virtool/virtool:15.1.1
           imagePullPolicy: Always
           name: virtool-api-jobs
-          command: ["python", "run.py", "jobsAPI", "--host=0.0.0.0"]
+          command: ["python", "run.py", "--no-tasks", "jobsAPI", "--host=0.0.0.0"]
```

Comment:
Does the jobsAPI run tasks? Thinking this change might be redundant

Otherwise this looks good to me